lnwallet/btcwallet/btcwallet.go

```diff
@@ -1203,7 +1203,9 @@ func mapRpcclientError(err error) error {
 	// If the wallet reports a double spend, convert it to our internal
 	// ErrDoubleSpend and return.
 	case errors.Is(err, chain.ErrMempoolConflict),
-		errors.Is(err, chain.ErrMissingInputs):
+		errors.Is(err, chain.ErrMissingInputs),
+		errors.Is(err, chain.ErrTxAlreadyKnown),
+		errors.Is(err, chain.ErrTxAlreadyConfirmed):
```

Comment:
Not sure if I'd consider `ErrTxAlreadyConfirmed` as double spend. Is there a reason to do so?

Reply:
So during startup, we may rebroadcast our force close tx that can cause this error, which matches the issue voltage has. I think previously we also considered this as `ErrDoubleSpend` [here](https://github.com/btcsuite/btcwallet/pull/918/files#diff-fde7eecc7d486a2d684192946f015f42f69af8aa27e10c0a019f0f3bebb28deeL4142).

Reply:
Maybe add a comment here why we classify a already confirmed tx as doublespend ?

Reply:
tbh I don't think it should be considered as a double spend - we are doing it to keep the existing behavior. IMO it's better to check whether the to-be-published tx already exists in the blockchain or in the mempool before broadcasting it during startup. Or even better, understand why it's broadcast again as I believe there are some internal state updates once a tx is published, but it looks like that state was not persisted.